docs/source/getting_started/content/proc_accessing-the-openshift-cluster-with-oc.adoc

```diff
@@ -50,4 +50,4 @@ See <<troubleshooting-codeready-containers_{context}>> if you cannot access the
 .Additional resources
 
 * The link:https://docs.openshift.com/container-platform/latest/applications/projects/working-with-projects.html[OpenShift documentation] covers the creation of projects and applications.
-* You can also use link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/understanding-odo.html[OpenShift Do] (`odo`) to create OpenShift projects and applications from the command line.
+* You can also use link:https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.html[OpenShift Do] (`odo`) to create OpenShift projects and applications from the command line.
```

Comment:
This URL works and redirects to the 4.3 docs: https://docs.openshift.com/container-platform/latest/cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.html

Reply:
I'm mostly thinking about this from a maintenance perspective.

Likewise, could we move this URL into an attribute defined in `docs/source/meta/product-attributes.adoc`? Something along the lines of `{odo-docs-url}`? This way we can redefine the URL when publishing on the CP to point to the CP copy of the `odo` docs.